src/widget/loginscreen.cpp

```diff
@@ -40,7 +40,7 @@ LoginScreen::LoginScreen(QWidget *parent) :
     ui->setupUi(this);
 
     // permanently disables maximize button https://github.com/tux3/qTox/issues/1973
-    this->setWindowFlags(windowFlags() &! Qt::WindowMaximizeButtonHint);
+    this->setWindowFlags(windowFlags() &~ Qt::WindowMaximizeButtonHint);
```

Comment:
Ah, damn, got me. You're right! So yes, I agree in this case, as the `!` "converts" to `bool`, while the `~` returns the bitwise negation.